sources/packages/backend/apps/queue-consumers/src/services/cas-invoice-batch/cas-invoice-batch.service.ts

```diff
@@ -57,8 +61,10 @@ export class CASInvoiceBatchService {
       if (!pendingReceipts.length) {
         processSummary.info("No pending receipts found.");
         // Cancels the transaction to rollback the batch number.
-        await entityManager.queryRunner.rollbackTransaction();
```

Comment:
Is the rollback not working as expected? - what is the reason for this change?

Reply:
It never worked as expected. Throwing an exception is the way to abort the transaction when not using the query runner way. So either we change to use the query runner (I was not willing to) or we use an exception.